playgrounds/angular/src/app/app.component.ts

```diff
@@ -100,7 +100,7 @@ import {
 export class AppComponent implements OnInit, AfterViewInit {
   @ViewChild(DxPopoverComponent) popover: DxPopoverComponent;
   
-  value: any = [
+  value: Array<Date|Number|String> = [
```

Comment:
Why so? You assign only Date values. Is there any other reason why use union of types instead of `Array<Date>`?

Reply:
This variable is used by the Calender.value option: https://js.devexpress.com/Angular/Documentation/ApiReference/UI_Components/dxCalendar/Configuration/#value

The type should be
` Date | Number | String | Array<Date | Number | String>`

but I just used an array of union types just in case someone that would use the example would like to use string or number values.